apps/studio/components/interfaces/Settings/General/CustomDomainConfig/CustomDomainActivate.tsx

```diff
@@ -158,7 +158,10 @@ const CustomDomainActivate = ({ projectRef, customDomain }: CustomDomainActivate
         onCancel={() => setIsActivateConfirmModalVisible(false)}
         onConfirm={onActivateCustomDomain}
       >
-        <p className="text-sm">The existing Supabase subdomain will be deactivated.</p>
+        <p className="text-sm">
+          This will activate the custom domain <code>{customDomain.hostname}</code>. Your project's
```

Comment:
nit: language is kinda repeating the title here, wondering if we can use the language from our docs site [here](https://supabase.com/docs/guides/platform/custom-domains#activate-your-domain)

Once activated, Supabase will serve the requests from your new domain. Your Supabase project domain will still continue to work and serve requests.